src/MetricsUploader/MetricsUploaderWindows.cpp

```diff
@@ -20,21 +20,17 @@ constexpr const char* kClientLogFileSuffix = "Orbit";
 
 MetricsUploader::MetricsUploader(std::string client_name, std::string session_uuid,
                                  Result (*send_log_event_addr)(const uint8_t*, int),
-                                 Result (*shutdown_connection_addr)(),
-                                 HMODULE metrics_uploader_client_dll)
+                                 Result (*shutdown_connection_addr)())
     : client_name_(std::move(client_name)),
       session_uuid_(std::move(session_uuid)),
       send_log_event_addr_(send_log_event_addr),
-      shutdown_connection_addr_(shutdown_connection_addr),
-      metrics_uploader_client_dll_(metrics_uploader_client_dll) {}
+      shutdown_connection_addr_(shutdown_connection_addr) {}
```

Comment:
I believe the HMODULE should stay here, even if we currently don't need it.

This class acts as the owner of that handle and we shouldn't throw it away.

Reply:
ok, thanks. I've left the HMODULE as a class member